src/html/write_inventory.jl

```diff
@@ -72,6 +75,29 @@ function write_inventory(doc, ctx)
 end
 
 
+function _find_project_version(doc, ctx)
+    version = ctx.settings.version
+    if isempty(version)
+        current_dir = pwd()
```

Comment:
I also considered starting from `doc.user.source` (the `docs/src` directory). But I think the current working directory (which I think Documenter changes to `docs`) will work just fine for pretty much any project, and saves one iteration of the loop.